raaghu-elements/src/rds-plan-switcher/rds-plan-switcher.tsx

```
@@ -0,0 +1,42 @@
+            id="1"
+          >
+            Free
+          </div>
+          <div 
+            onClick={() => setIsPlanFree(false)} 
+            className={`equalWidth ${!isPlanFree ? "premiumSelected" : ""}`} 
+            id="2"
+          >
+            <div className={`premiumContent ${isPlanFree ? 'free' : ''}`}>
+              <img 
+                src={isPlanFree ? "./assets/crown.png" : "./assets/crownwhite.png"} 
```

ESLint

[error] Expected indentation of 24 spaces but found 16.

(indent)